common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -529,6 +529,33 @@ public static final class AAD {
         public static final String APP_VERSION = "x-app-ver";
     }
 
+    /**
+     * Represents the constants value for the SwitchBrowser protocol.
+     */
+    @NoArgsConstructor(access = AccessLevel.PRIVATE)
+    public static final class SWITCH_BROWSER {
+
+        /**
+         * String Query parameter key to indicate support for SWITCH_BROWSER protocol.
+         */
+        public static final String SWITCH_BROWSER = "switch_browser";
+
+        /**
+         * String Query parameter key for the session token.
+         */
+        public static final String CODE = "code";
+
+        /**
+         * String Query parameter key for the switchBrowser action uri.
+         */
+        public static final String ACTION_URI = "action_uri";
+
+        /**
+         * String Query parameter key for the switchBrowser action.
+         */
+        public static final String ACTION = "action";
```

Comment:
We've now removed this on the server side. An action query param is no longer sent to the client.

Reply:
removed